cypress/e2e/HomepageNavigation/header_and_footer_func.js

```diff
@@ -0,0 +1,122 @@
+/// <reference types="cypress" />
+const { url, homePage, textInPages } = require('../../support/selectors')
+
+describe('Test header and footer of homepage', () => {
+
+    it('Test Case 5. Test BBC homepage logo', () => {
+        cy.get(homePage.newsButtons).eq(0).click({force: true});
+        cy.url().should('eq', url.newsPage);
+        cy.get(homePage.logo).eq(0).click({force: true});
+        cy.url().should('eq', url.landingPage);
+    });
+
+    it('Test Case 6. Test BBC homepage hamburger', () => {
+        cy.get(homePage.hamburger).click({force: true});
+        cy.get(homePage.weatherButtons).eq(1).should('exist');
+    });
+
+    it('Test Case 7. Test BBC homepage registration button', () => {
+        cy.get(homePage.registerButton).click({force: true});
+        cy.contains(textInPages.agePageVerificationText).should('be.visible');
+
+    });
+
+    it('Test Case 8. Test BBC homepage sign in button', () => {
+        cy.get(homePage.signInButton).click({force: true});
+        cy.contains(textInPages.signInPageVerificationText).should('be.visible');
+
+    });
+
+    it('Test Case 9. Footer Homepage, News, Sport Business and Innovation button test', () => {
+        cy.get(homePage.newsButtons).eq(1).scrollIntoView().click({force: true});
```

Comment:
ar scroll into view yra butinas ?>

Reply:
Cypress kaip ir supranta kad reikia nueit taciau kartais buna uzgrybauja, tad just to make sure pridejau